setup.py

```diff
@@ -89,8 +92,18 @@
             "prophet>=1.0.1",
             "statsmodels>=0.12.2",
             "hcrystalball==0.1.10",
+            "pytorch-forecasting>=0.9.0",
+            "pytorch-lightning==1.5.10",
+            "tensorflow>=2.5.1",
```

Comment:
why are they required here? If they are pytorch-forecasting's dependencies, they should be installed automatically. If they are not, please provide reasons.